### ISSUE TITLE
Bull's eye maculopathy

### ISSUE THREAD

**Issue by Sukramg** [obophenotype/human-phenotype-ontology#10847]
**Preferred term label:**
Bull's eye maculopathy

**Synonyms**
Foveal dystrophy
annular macular dystrophy
Cone dystrophy (if associated with cone cell dysfunction)
Toxic maculopathy (when caused by drug toxicity, e.g., chloroquine or hydroxychloroquine)

**Definition (free text, please give PubMed ID)**
PMID: [31704701](https://pubmed.ncbi.nlm.nih.gov/31704701/)

**Parent term (use hpo.jax.org/app)**
Macular dystrophy HP:0007754

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
# 153870

**Your nano-attribution (ORCID)**
0000-0002-6601-4486

**Comment by daboe01** [comment]
see [11505](https://hpo.jax.org/browse/term/HP:0011504)

**Comment by pnrobinson** [comment]
See existing term
Bull's eye maculopathy HP:0011504
I will add the above a synonyms.

**Comment by pnrobinson** [comment]
added synonyms.
